### PR TITLE
First round of cli features completed.

### PR DESCRIPTION
**Pull request by etiennenoel**

### CODE REVIEW

File: packages/validation/src/interceptors/body-validation.request-interceptor.ts [hunk 1]
```diff
@@ -44,7 +46,7 @@ export class BodyValidationRequestInterceptor implements RequestInterceptorInter
         }, ValidationModuleKeyname)
 
         // Validates that the body can be mapped to the expected type
-        const mappedBody = plainToClass(methodNode.route.context.bodyValidator.classType, request.body);
+        const mappedBody = plainToInstance(methodNode.route.context.bodyValidator.classType, request.body);
```
Comment (mathieugh) [inline packages/validation/src/interceptors/body-validation.request-interceptor.ts:49]: What's the difference here ?

File: tests/cli/src/launch-command.scenario.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import {exec} from "child_process";
+
+describe('CLI - Launch command',  () => {
+    it("should properly launch a command", () => {
+        // Launch the sample command
+        // todo: make sure the help and list commands can be successfully launched.
```
Comment (mathieugh) [inline tests/cli/src/launch-command.scenario.e2e.ts:6]: This does not seem to be done ?

File: tests/e2e/scenarios/modules/cli/sample.command.js
Comment (mathieugh) [inline tests/e2e/scenarios/modules/cli/sample.command.js:1]: Do we want to commit the JS file ?

**Review by mathieugh — CHANGES_REQUESTED** [review]

File: packages/validation/src/interceptors/body-validation.request-interceptor.ts (hunk 1 above)
Comment (etiennenoel) [reply packages/validation/src/interceptors/body-validation.request-interceptor.ts:49]: plainToClass is deprecated

File: packages/cli/readme.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+Then, it will validate that the arguments passed are valid.
+
+If yes, then the `run` method of the Command with the proper arguments.
+
+
+#Latest to do
+Write a new regex that captures the parameters and their values in separate group
+
+# How to use
```
[the remark this answers is not in the document]
Comment (etiennenoel) [reply packages/cli/readme.md:20]: Done.

File: tests/cli/src/launch-command.scenario.e2e.ts (hunk 2 above)
Comment (etiennenoel) [reply tests/cli/src/launch-command.scenario.e2e.ts:6]: Done.

**Review by mathieugh — APPROVED** [review]

**Pull request merged**